packages/website/ts/types.ts

```diff
@@ -361,6 +361,7 @@ export enum WebsitePaths {
     Instant = '/instant',
     Ecosystem = '/eap',
     MarketMaker = '/market-maker',
+    Governance = '/governance',
```

Comment:
We decided to change this to `/vote` and have dropped a landing page there currently.